### ISSUE TITLE
[Bug] ttnn.lgamma: z=0.75 orphaned by strict `<` guards on both adjacent Taylor bridges

### ISSUE THREAD

**Issue by kinginu** [tenstorrent/tt-metal#53046]
**Component / Area:** kernels (LLK/SFPU)

**Issue Type:** Bad Outputs

## Observed

`ttnn.lgamma` is 54x less accurate at `z = 0.75` than at the fp32 value one step above it.

Blackhole P300, fp32, absolute error against `torch.lgamma` in float64:

| z | error |
|---|---|
| **0.75** | **2.8510e-03** |
| 0.7500001 | 5.2661e-05 |

`z = 1.75` shows the same shape, 3.3475e-06 against 7.9167e-07 at 1.7500001.

**This is not #51976.** Inputs *below* 0.75 are also wrong — 3.1585e-03 at 0.74, 4.8202e-03 at 0.70, 2.9493e-02 at 0.55 — and that region is what #51976 reports and #51991 fixes with a new bridge on `[0.5, 0.75)`. `z = 0.75` itself is outside that range and has a different cause, and it survives that fix. See below.

## Expected

Accuracy that does not step across a single fp32 value.

## Steps to reproduce

```python
import torch, ttnn

d = ttnn.open_device(device_id=0)
vals = torch.tensor([0.75, 0.7500001, 1.75, 1.7500001], dtype=torch.float32)
x = torch.zeros((1, 1, 32, 32), dtype=torch.float32)
x.view(-1)[: vals.numel()] = vals

t = ttnn.from_torch(x, dtype=ttnn.float32, layout=ttnn.TILE_LAYOUT, device=d)
got = ttnn.to_torch(ttnn.lgamma(t)).view(-1)[: vals.numel()].double()
ref = torch.lgamma(vals.double())

for v, g, r in zip(vals.tolist(), got.tolist(), ref.tolist()):
    print(f"z={v:<12g} device={g:<18.10g} ref={r:<18.10g} abs_err={abs(g - r):.4e}")
ttnn.close_device(d)
```

On `1227e18213e`, clean build, nothing applied:

```
z=0.75        device=0.2004299462     ref=0.2032809514     abs_err=2.8510e-03
z=0.7500001   device=0.2032283        ref=0.2032810        abs_err=5.2661e-05
z=1.75        device=-0.08440446854   ref=-0.08440112102   abs_err=3.3475e-06
z=1.7500001   device=-0.0844003       ref=-0.0844011       abs_err=7.9167e-07
```

Bit-for-bit reproducible.

## Which inputs are affected

The fp32 arm has two Taylor bridges, guarded at `ckernel_sfpu_lgamma.h:126` and `:136`, on `|z-1|` and `|z-2|` against 0.25. Four fp32 values sit exactly on a bound. All four take the fall-through path; two of them are visibly wrong there and two are not:

| z | on the bound of | error | one step inside the bridge | |
|---|---|---|---|---|
| 0.75 | `\|z-1\|` | 2.8510e-03 | 5.2661e-05 | **54.1x** |
| 1.25 | `\|z-1\|` | 5.2284e-05 | 3.4176e-05 | 1.5x |
| 1.75 | `\|z-2\|` | 3.3475e-06 | 7.9167e-07 | **4.2x** |
| 2.25 | `\|z-2\|` | 4.7583e-07 | 4.7583e-07 | 1.0x |

What separates them is where the fall-through lands. It is the Stirling arm, an asymptotic expansion for large `z`, and `z = 0.75` is the furthest of the four from where that expansion is any good.

## The interaction with #51991

That PR adds a bridge around `z = 0.625` guarded by `v_if(z < 0.75f)`, ahead of the existing `v_elseif(abs_range1 < 0.25f)`. Both comparisons are strict and both stop at 0.75:

* `z < 0.75f` is false at `z = 0.75`
* `|0.75 - 1| = 0.25` is not `< 0.25`

So after that PR, `z = 0.75` is still sent to Stirling, and it becomes a one-point hole between two adjacent bridges rather than the edge of a wide one. Read from the diff; not measured under that PR.

## Why no test catches it

`tests/ttnn/unit_tests/operations/eltwise/test_math.py::test_lgamma` draws random inputs and checks PCC at 0.99. The affected inputs are four individual fp32 values that random sampling does not hit, and one bad element out of `h*w` does not move PCC.

## Environment

* OS: Ubuntu 22.04
* Version of software: tt-metal `1227e18213e`, clean build, nothing applied
* Hardware: Blackhole P300, device 0

## Not established

* **Wormhole.** Its `ckernel_sfpu_lgamma.h` is not identical to Blackhole's — Blackhole carries an extra `calculate_lgamma_adjusted` — but the two guards are the same, at `:87` and `:97`. Not measured; there is no Wormhole hardware on this host.
* **bfloat16.** Everything above is the fp32 arm. The bf16 path has its own structure and was not swept.
* **Whether this is a pattern elsewhere.** `ckernel_sfpu_trigonometry.h:1118` guards a fit its own comment describes as being on the closed `[0, 0.75]` with a strict `<`. Sweeping `asinh` across that bound: the series side reaches 0.85 fp32 ULP, the single fall-through value 0.02, and the `log1p` side 0.95 — so the strict guard costs nothing there, because that fall-through is as accurate as the series. That is one op checked, not a survey.
